### PR TITLE
Merges currently published version into "master"

### PR DESCRIPTION
**Pull request by artem-zakharchenko**
This pull request's purpose is to merge currently published version of the package to the `master` branch.

## Motivation

- To be evident what code is actually published
- To be updated with the master branch (feature branch has some commits behind)
- To be able to continue development

### CODE REVIEW

**Comment by artem-zakharchenko** [comment]
The latest failed status of the CI alerts that AppVeyor is not configured properly:

```
Build started
git clone -q --branch=kubula/customize_json_converter_html https://github.com/apiaryio/gavel2html.git C:\projects\gavel2html
git checkout -qf dd4d441efa04d948ba2009a83f70910ba3871c51
Specify a project or solution file. The directory does not contain a project or solution file.
```

**Comment by honzajavorek** [comment]
Yeah, could you copy-paste config from dredd or dredd-transactions or elsewhere?

**Comment by artem-zakharchenko** [comment]
@honzajavorek no need, I've resolved the configs. Do you find them okay?

**Comment by honzajavorek** [comment]
Looks okay 👍

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "gavel2html",
-  "version": "0.0.0-semantically-released",
-  "description": "Render Gavel.js output to HTML diff",
+  "version": "0.2.0-unstable.1",
```
Comment (honzajavorek) [inline package.json:3]: As a next step, I'd bump this in a separate PR to `0.2.0-unstable.2` or `0.2.1`, release, and upgrade in the core app, so we see this is really the code we think it is 🙈

**Review by honzajavorek — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (artem-zakharchenko) [reply package.json:3]: Linked the package and tried in the core app. Confirms working without changes.

**Pull request merged**